### PR TITLE
GivBack design for Single Project Page

### PR DESCRIPTION
**Pull request by RamRamez**
#684

### CODE REVIEW

File: src/components/views/project/projectDonateCard/GIVbackToast.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	Subline,
+} from '@giveth/ui-design-system';
+import styled from 'styled-components';
+import { Flex, FlexCenter } from '@/components/styled-components/Flex';
+import useDetectDevice from '@/hooks/useDetectDevice';
+
+const GIVbackToast = () => {
+	const [isPopupOpen, setIsPopupOpen] = useState(false);
+	const { isMobile } = useDetectDevice();
+	const togglePopup = () => setIsPopupOpen(!isPopupOpen);
```
Comment (mateodaza) [inline src/components/views/project/projectDonateCard/GIVbackToast.tsx:16]: Can we close the toast when user clicks outside just like all modals currently work? I believe it's better UX

**Review by mateodaza — CHANGES_REQUESTED** [review]
Awesome work @RamRamez just a little UX detail feedback

**Comment by MoeNick** [comment]
@mateodaza this design provided by @Tosinolawale and I thinks its good we fix this toast and don't let the user close it. We wanna emphasize on GIVbacks

**Comment by mateodaza** [comment]
> @mateodaza this design provided by @Tosinolawale and I thinks its good we fix this toast and don't let the user close it. We wanna emphasize on GIVbacks

image

I meant the tooltip :) used the wrong word

**Comment by MohammadPCh** [comment]
@RamRamez @mateodaza please check the IconWithTooltip component. maybe it helps you.

**Review by mateodaza — APPROVED** [review]
Thanks for your changes!

**Pull request merged**